bigframes/core/indexes/base.py

```diff
@@ -115,6 +115,20 @@ def from_frame(
         index._linked_frame = frame
         return index
 
+    def to_frame(
```

Comment:
Plz add tests.

Reply:
done in tests/unit/core/test_indexes.py :D